### ISSUE TITLE
Error when resuming training

### ISSUE THREAD

**Issue by PiyushM1** [Tessellate-Imaging/monk_v1#88]
![Screenshot 2020-07-28 at 9 01 05 PM](https://user-images.githubusercontent.com/52419808/88687148-88ed6880-d115-11ea-9303-3f7b33ecb4a2.png)
![Screenshot 2020-07-28 at 9 01 19 PM](https://user-images.githubusercontent.com/52419808/88687157-8c80ef80-d115-11ea-8699-3f87ca06d8a4.png)
![Screenshot 2020-07-28 at 9 01 31 PM](https://user-images.githubusercontent.com/52419808/88687182-93a7fd80-d115-11ea-9451-5ec50ad1fde8.png)

After training for 5 epochs, I decided to update the learning rate and resume training for another 5 epochs. This threw some exceptions. Though I have resumed exactly how it is shown in the documentation, if I am making a mistake, then let me know.

**Comment by abhi-kumar** [comment]
Thank you for pointing out the error.

**Comment by abhi-kumar** [comment]
The issue is with newer versions of keras, which has updated history keys from val_acc to val_accuracy. It has been resolved now. Please check and let us know

**Comment by PiyushM1** [comment]
It did not work for me.
![Screenshot 2020-08-01 at 8 02 46 PM](https://user-images.githubusercontent.com/52419808/89103788-168cc900-d432-11ea-9efb-711e11bf885d.png)
![Screenshot 2020-08-01 at 8 03 05 PM](https://user-images.githubusercontent.com/52419808/89103790-18ef2300-d432-11ea-84ca-049ff4935a71.png)

**Comment by abhi-kumar** [comment]
Thank you for letting us know, the error will be resolved soon
